example/android/gradle/wrapper/gradle-wrapper.properties

```
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
```
I would suggest to update to the latest version.

---

yup!